packages/overlay/src/vaadin-overlay.d.ts

```diff
@@ -197,6 +202,19 @@ declare class Overlay extends ThemableMixin(DirMixin(ControllerMixin(HTMLElement
     listener: (this: Overlay, ev: OverlayEventMap[K]) => void,
     options?: EventListenerOptions | boolean,
   ): void;
+
+  protected _finishClosing(): void;
```

Comment:
This method is used by `vaadin-confirm-dialog` to fire an event.
We could consider adding an event instead, I'll create a separate PR.